tests/ui/check-cfg/allow-same-level.stderr

```diff
@@ -4,7 +4,7 @@ warning: unexpected `cfg` condition name: `FALSE`
 LL | #[cfg(FALSE)]
    |       ^^^^^
    |
-   = help: expected names are: `clippy`, `debug_assertions`, `doc`, `doctest`, `fmt_debug`, `miri`, `overflow_checks`, `panic`, `proc_macro`, `relocation_model`, `rustfmt`, `sanitize`, `sanitizer_cfi_generalize_pointers`, `sanitizer_cfi_normalize_integers`, `target_abi`, `target_arch`, `target_endian`, `target_env`, `target_family`, `target_feature`, `target_has_atomic`, `target_has_atomic_equal_alignment`, `target_has_atomic_load_store`, `target_os`, `target_pointer_width`, `target_thread_local`, `target_vendor`, `test`, `ub_checks`, `unix`, and `windows`
+   = help: expected names are: `clippy`, `debug_assertions`, `doc`, `doctest`, `emscripten_wasm_eh`, `fmt_debug`, `miri`, `overflow_checks`, `panic`, `proc_macro`, `relocation_model`, `rustfmt`, `sanitize`, `sanitizer_cfi_generalize_pointers`, `sanitizer_cfi_normalize_integers`, `target_abi`, `target_arch`, `target_endian`, `target_env`, `target_family`, `target_feature`, `target_has_atomic`, `target_has_atomic_equal_alignment`, `target_has_atomic_load_store`, `target_os`, `target_pointer_width`, `target_thread_local`, `target_vendor`, `test`, `ub_checks`, `unix`, and `windows`
```

Comment:
The `emscripten_wasm_eh` cfg name is needed as an implementation detail for `panic_unwind` and `unwind`. If there is some way to expose it only in the standard library and keep it out of the help messages that might be preferable but I didn't see any way to do that.

Reply:
No implementation detail should leak onto user-facing diagnostics, you need to remove the `ins!(sym::emscripten_wasm_eh, no_values);` from the `CheckCfg::fill_well_known` and instead "manually" add the cfg to the check-cfg section of the `Cargo.toml` of [`panic_unwind`](https://github.com/rust-lang/rust/blob/ac00fe89a121e9ca5e59c093c7c02aa296e55ece/library/panic_unwind/Cargo.toml#L26) and [`unwind`](https://github.com/rust-lang/rust/blob/ac00fe89a121e9ca5e59c093c7c02aa296e55ece/library/unwind/Cargo.toml#L40).

Something like this:
```diff
- check-cfg = []
+ check-cfg = ['cfg(emscripten_wasm_eh)']
```

Reply:
Thanks!